.circleci/config.yml

```diff
@@ -52,7 +53,8 @@ install_singularity: &install_singularity
     sudo make install
     singularity --version
 # Jobs
-version: 2
+orbs:
+  singularity: singularity/singularity@1.0.3
```

Comment:
OK that this is different singularity version from what pipeline runs?

Reply:
1.0.3 here refers to the version of the singularity orb. See more: https://circleci.com/orbs/registry/orb/singularity/singularity

Reply:
Ah OK